src/sparse_blas/backends/mkl_common/mkl_operations.cxx

```diff
@@ -71,21 +76,32 @@ std::enable_if_t<detail::is_fp_supported_v<fpType>, sycl::event> gemv(
 }
 
 template <typename fpType>
-std::enable_if_t<detail::is_fp_supported_v<fpType>> trsv(sycl::queue& /*queue*/, uplo /*uplo_val*/,
-                                                         transpose /*transpose_val*/,
-                                                         diag /*diag_val*/,
-                                                         detail::matrix_handle* /*A_handle*/,
-                                                         sycl::buffer<fpType, 1>& /*x*/,
-                                                         sycl::buffer<fpType, 1>& /*y*/) {
-    throw unimplemented("SPARSE_BLAS", "trsv");
+std::enable_if_t<detail::is_fp_supported_v<fpType>> trsv(sycl::queue& queue, uplo uplo_val,
+                                                         transpose transpose_val, diag diag_val,
+                                                         detail::matrix_handle* A_handle,
+                                                         sycl::buffer<fpType, 1>& x,
+                                                         sycl::buffer<fpType, 1>& y) {
+    if (transpose_val != transpose::nontrans) {
+        throw mkl::unimplemented("sparse_blas/backends/mkl", __FUNCTION__,
+                                 "Transposed or conjugate trsv is not supported");
+    }
```

Comment:
Add `TODO` comment:
```suggestion
    // TODO: Remove this if condition once Intel oneMKL adds support for
    // trans/conjtrans to trsv
    if (transpose_val != transpose::nontrans) {
        throw mkl::unimplemented("sparse_blas/backends/mkl", __FUNCTION__,
                                 "Transposed or conjugate trsv is not supported");
    }
```

Reply:
Done in https://github.com/oneapi-src/oneMKL/pull/407/commits/e49f285b7e41f354c6423359238ce2ace2d9b50d